src/server/controllers/users.controller.js

```diff
@@ -21,22 +21,19 @@ export function getAll(req, res, next) {
 
 // Get all user list except logged in user
 export function getMentorList(req, res, next) {
-  // TODO: Once session_auth is implemented, replace the email address with req.session.email.
-  User.find({ 'email' : { $ne : 'session@yoda.com' }},
-    // TODO: Longer term, we should migrate to a UserSummary object
-    // that contains subset of fields. For now, we return only the following
-    // fields from user object:
-    //
-    // - userId
-    // - name
-    // - field
-    { userId: 1, email: 0, name: 1, age: 0, field: 1, region: 0, skills: 0, regDate: 0 }, (err, doc) => {
+  if (typeof req.session.access_token != 'undefined' && req.session.access_token == req.query.access_token) {
```

Comment:
break line after &&.
Also, I think the styleguide states we should use === and !== over == and != respectively. I haven't been paying attention to it till now, but we should start doing so. :)

See https://github.com/airbnb/javascript#comparison--eqeqeq